dist-unminified/recorder.js

```diff
@@ -197,7 +201,9 @@ Recorder.prototype.initSourceNode = function( sourceNode ){
   var self = this;
   return global.navigator.mediaDevices.getUserMedia({ audio : this.config.mediaTrackConstraints }).then( function( stream ){
     self.stream = stream;
-    return self.audioContext.createMediaStreamSource( stream );
+    self.microphone = self.audioContext.createMediaStreamSource( stream );
```

Comment:
What about users that choose to use the sourceNode option in `start`? I think I would prefer a feature where the   gain for the input sourceNode is adjustable, not just for a microphone.

Reply:
Perhaps we could rename it to `recordingGain` instead of `microphoneGain`